sys-libs/nss-usrfiles/nss-usrfiles-9999.ebuild

```diff
@@ -9,7 +9,7 @@ CROS_WORKON_REPO="git://github.com"
 if [[ "${PV}" == 9999 ]]; then
 	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
 else
-	CROS_WORKON_COMMIT="42bec47544ad80d3e39342b11ea33da05ff9133d" # v2.23.0
+	CROS_WORKON_COMMIT="9078c543ba7d2bc5011737675b3dddb882673ce7" # flatcar-master
```

Comment:
This commit ID is from `master`, not `flatcar-master`. Both branches are the same, though…

Reply:
I'll fix this and use the one from flatcar-master.